pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTableRestletResource.java

```diff
@@ -697,6 +698,31 @@ public RebalanceResult rebalance(
     }
   }
 
+  /** Waits for jobId to be persisted using a retry policy. Tables with 100k+ segments
+    * take up to a few seconds for the jobId to persist. This ensures the jobId is present
+    * before returning the jobId to the caller, so they can correctly poll the jobId. **/
+  public void waitForJobIdToPersist(String jobId, String tableNameWithType) {
+      try {
```

Comment:
(code format) Can you set up [Pinot Style](https://docs.pinot.apache.org/developers/developers-and-contributors/code-setup#set-up-ide) and reformat the change